idl/cosmo-spd.idl

```diff
@@ -0,0 +1,14 @@
+// Cosmo SPD task API
+
+Interface(
```

Comment:
I'm not quite sure what the purpose of this interface is, I think it's supposed to be a placeholder?